### ISSUE TITLE
I was able to get a 32-bit build to pass

### ISSUE THREAD

**Issue by Keithcat1** [synthizer/synthizer#14]
After some fiddling around I got a 32-bit build to pass on my fork at https://github.com/keithcat1/synthizer
The reason I'm not doing a PR is because I'm not sure how to get it to build both 32 and 64-bit binaries at the same time
The problem seems to be that CMAKE isn't telling clang to build and link in 32-bit mode, so I added these environment variables:
cflags: -m32
cxxflags: -m32
ldflags: -machine:x86
The rest was fairly trivial, using 32-bit Python instead of the 64-bit version.
I didn't upload any wheels though.

**Comment by camlorn** [comment]
Ok, cool. I'll take a look at getting this merged into the CI infrastructure.  Thanks for hopefully unblocking this work.

**Comment by camlorn** [comment]
I've folded these changes in and pushed a 0.7.1 release that should have working 32-bit wheels, but am not currently set up with a 32-bit environment to test them in; can you validate whether they work and close this issue if that's the case?

**Comment by Keithcat1** [comment]
Works. Pip install synthizer worked, and I was able to play a sound as well. This is using my 32-bit computer.
